oci/layout/oci_transport.go

```diff
@@ -228,7 +228,16 @@ func (ref ociReference) NewImageDestination(ctx context.Context, sys *types.Syst
 
 // DeleteImage deletes the named image from the registry, if supported.
 func (ref ociReference) DeleteImage(ctx context.Context, sys *types.SystemContext) error {
-	return errors.New("Deleting images not implemented for oci: images")
+	index := filepath.Join(ref.dir, "index.json")
```

Comment:
Please use existing utilities like `indexPath()`, so that references to each file all go through a single code path and can be trivially located.